cmd/bosun/web/static/js/bosun.js

```diff
@@ -103,18 +103,12 @@ bosunControllers.controller('BosunCtrl', ['$scope', '$route', '$http', '$q', '$r
     $scope.panelClass = function (status, prefix) {
         if (prefix === void 0) { prefix = "panel-"; }
         switch (status) {
-            case "critical":
```

Comment:
I'm aware that typescript is doing this, but can you remove this file from the PR?